draft-ounsworth-cfrg-kem-combiners.mkd

```diff
@@ -228,40 +251,55 @@ ss = kemCombiner(ss_1, ss_2, ..., ss_n)
 
 This document assumes that shared secrets are the output of a KEM, but without loss of generality they MAY also be any other source of cryptographic key material, such as pre-shared keys (PSKs), with PQ/PSK being a quantum-safe migration strategy being made available by some protocols, see for example IKEv2 in [RFC8784].
 
-In general it is desirable to use a multi-PRF as a KEM combiner, meaning a PRF when keyed by any of its single inputs.
+In general it is desirable to use a multi-PRF as a KEM combiner, meaning that the combiner has the properties of a PRF when keyed by any of its single inputs.
 The following simple yet generic construction can be used in all IETF protocols that need to combine the output of two or more KEMs:
 
 ~~~
 KDF(counter || k_1 || ... || k_n || fixedInfo, outputBits)
 ~~~
-{: title="general KEM combiner construction" }
+{: #tab-kemCombiner title="general KEM combiner construction" }
 
 where:
 
 - `KDF` represents a suitable choice of cryptographic key derivation function,
-- `k_i` represent the constant-length input keys,
+- `k_i` represent the constant-length input keys and is discussed in {{sec-k_i-construction}},
 - `fixedInfo` is some protocol-specific KDF binding,
 - `counter` parameter is instantiation-specific and is discussed in {{sec-instantiation}}.
-- `outputBits` determines the length of the key,
+- `outputBits` determines the length of the key to output,
 - `||` represents concatenation.
 
-In section {{sec-instantiation}} are listed several possible practical instantiations, in compliance with NIST SP-800 56Cr2 {{SP800-56C}}.
-The shared secret `ss` MAY be used as a Key Encryption Key (KEK) for key encapsulation.
+In {{sec-instantiation}} are listed several possible practical instantiations, in compliance with NIST SP-800 56Cr2 {{SP800-56C}}.
+The shared secret `ss` MAY be used directly as a symmetric key, for example as a MAC key or as a Key Encryption Key (KEK) for key encapsulation.
```

Comment:
```suggestion
The shared secret `ss` MAY be used directly as a symmetric key, for example as a MAC key or as a Key Encryption Key (KEK).
```
IMO the last words "for key encapsulation" should be removed since there you tend to mean key encapsulation with a KEK (so mostly symmetrical) which is in conflict with the key encapsulation via KEMs. I think ending with KEK is fine.